src/manager.js

```diff
@@ -2,17 +2,21 @@ require('./dfp');
 var utils = require('./utils');
 var TargetingPairs = require('./helpers/TargetingPairs');
 var AdZone = require('./helpers/AdZone');
-var PageDepth = require('./helpers/PageDepth');
 var Feature = require('./helpers/Feature');
 
 var ERROR = 'error';
 
 var AdManager = function (options) {
+  var PageDepth = require('./helpers/PageDepth');
```

Comment:
What's the motivation behind moving this?

Reply:
It's only used there, so I figured it makes sense to only define it within the context that it is used rather than having it exist at the global level. That being said, the likelihood that it would ever create an issue is incredibly minuscule ¯\_(ツ)_/¯

Reply:
What is the best practice? I've typically only done this in other languages to avoid circular dependencies.